aws/resource_aws_glue_crawler_test.go

```diff
@@ -1122,10 +1122,10 @@ resource "aws_glue_classifier" "test2" {
 resource "aws_glue_crawler" "test" {
   depends_on = ["aws_iam_role_policy_attachment.test-AWSGlueServiceRole"]
```

Comment:
Please note that quoted `depends_on` entries are also deprecated syntax, but we can always grab these in later syntax update passes if `terrafmt` is still returning errors on some blocks. 👍 

```suggestion
  depends_on = [aws_iam_role_policy_attachment.test-AWSGlueServiceRole]
```